src/rulesets/Base/array.jl

```diff
@@ -13,7 +13,7 @@ end
 function rrule(::typeof(reshape), A::AbstractArray, dims::Union{Colon,Int}...)
     A_dims = size(A)
     function reshape_pullback(Ȳ)
-        ∂A = reshape(Ȳ, A_dims)
+        ∂A = reshape(unthunk(Ȳ), A_dims)
```

Comment:
Does this not work? https://github.com/JuliaDiff/ChainRulesCore.jl/blob/653cd7d626183faf70107a6c639c04ae4a85f0c5/src/differentials/thunks.jl#L49

Reply:
```suggestion
        ∂A = reshape(Ȳ, A_dims)
```